### ISSUE TITLE
Feature request: Show common sources and work-arounds

### ISSUE THREAD

**Issue by jessehouwing** [report-uri/report-uri-bugs#40]
This may need some crowdsourcing, but it'd be very valuable for people trying to understand the source for certain hits:

* data:img that are inserted by certain browser plugins (Pinterest, Skype)
* Font-src that are inserted by certain browser plugins (Skype for example)
  * And linking to a credible source on disabling that behavior: http://bacsoftwareconsulting.com/blog/index.php/web-development/prevent-skype-from-highlighting-phone-numbers/
* For certain common services possibly detect partial implementations. E.g. Google Adwords and Google Ads requires a number of domains in a number of categories. If a blocked uri is detected that matches such service, show the correct implementation.

I'd also love the ability to fetch the CSP for that source URL and show the required change to make the blocked content work. Show it as a DIFF/Patch.

**Comment by mdmcginn** [comment]
Yes, I had to read several error messages to find out how to change my .htaccess enough  to get Google Fonts to load.
